src/libfuncs/uint8.rs

```diff
@@ -275,8 +275,8 @@ mod test {
     use crate::utils::test::run_cairo;
     use serde_json::json;
 
-    fn error_value() -> serde_json::Value {
-        json!([(), [1, [[], [608642104203229548495787928534675319u128]]]])
+    fn error_value(n: u128) -> serde_json::Value {
+        json!([(), [1, [[], [n]]]])
     }
```

Comment:
The idea of this helper function is to not have to type the huge number every time. Adding the number as a parameter defeats the purpose.

Maybe add two separate functions?

```rs
fn u8_add_overflow_error() { ... }
fn u8_sub_overflow_error() { ... }
```